### DISCUSSION TITLE
The new scheduler preset is far too complex  in 1.17

### DISCUSSION THREAD

**Discussion by Muskelmagier** [Acly/krita-ai-diffusion#638] [General]
The new Scheduler feature is far too complex for this Addon's intention. 

The best example is that Many checkpoints recommend a simple "Eula ancestral" but in the new presets that one is not even mentioned. I have to learn ComfiUI Sampler and scheduler now, so that I can create a custom preset to just use that one in the right configuration. 

The system from before was definitely easier to manage. 

And I would no longer recommend a beginner to get the 1.17 version but stay in 1.16

**Comment by Acly** [comment]
Well I don't mind extending the default presets. But apart from few missing entries it's not so different from before?

I don't think samplers are something a beginner should have to care about at all. I doubt people will configure it in the long run, it's rather a technical detail. Not quite there yet with various checkpoints/lora having special needs, but I don't expect that's something you start out with. Nor would I consider learning about Euler, DPM++, etc. to be a priority...

**Comment by motolo** [comment]
The new presets are extremely simple :) actually and give a lot of freedom :)
It gives you the possibility to create many presets for your needs and gives the community the opportunity to share their settings. So simply just read the wikis and study a little to understand the functionality :) Thanks Acly for the new settings:)
